src/occopt/memory.h

```diff
@@ -24,7 +24,7 @@
  */
 
 #include "ctypes.h"
-#ifdef BORLAND
+#if defined(BORLAND) || (defined(__MINGW32__) && defined(__clang__))
```

Comment:
Hm, what actually is buggy there and where should those functions be defined?

Reply:
@GitMensch See: https://github.com/LADSoft/OrangeC/issues/907#issuecomment-1634641363